notes.txt

```diff
@@ -52,4 +52,12 @@ end
 %form.col-md-5.navbar-form(action="")
     .form-group
       %input.form-control(type="text" placeholder="Search for videos here")
-    %button.btn.btn-default(type="submit") Search
\ No newline at end of file
+    %button.btn.btn-default(type="submit") Search
+
+
+
+
+    fabricate reviews and reviews are set as instance variables, order of reviews is reverse chronically ordered (most recent reviews at top), average rating for a video right under title which reflects average of ratings in the reviews (can keep to one decimal point)
+
+
+    write test first for controller, success path, the validation error path, review has to have content. not authorized path (if user is not signed in they cannot add a review)
```

Comment:
While I am complete favor of writing down useful notes throughout the project I would keep them out of the git for the project. The project should contain precisely what it needs and no more. If you purview github you will find many commits on projects that do nothing more than eliminate unneeded whitespace. Make every line justify its existence.